### ISSUE TITLE
A11y_.NETWPF_ExpenseItIntro_View Expense Report_ColorContrast: Color Contrast for the "view expense report" title and background is less than 4.5:1(3.981:1). 

### ISSUE THREAD

**Issue by Blessymanna** [dotnet/wpf#5043]
[Check out Accessibility Insights! ]([url](url))- Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:#A11yMAS;#A11ySev2;#A11yTCS;#A11yWCAG;#DesktopApp;#JAWS;#NVDA;#WCAG1.4.3;#DesktopApp;#Win10;#BM-.NetCore(WPF)-Win32-July2021;#.NET Core;#Benchmark;

### Environment details:
Application Name: .NET Core WPF
Microsoft .NET Core SDK Version 6.0.100-rc.1.21380.2
Windows Version: Windows10

### Repro Steps:

1. Launch VS 2019 Int Preview.
2. Navigate to Getting started and right click on the Expense it intro and navigate to build
3. Then navigate on Expense it Intro then Click on Debug-->Start New Instance.
4. Check whether the "view expense report" title and the background color contrast is 4.5:1 or not.
### Actual Result:
The "view expense report" title and its background color contrast is less than 4.5:1, i.e 3.981:1

### Note:
The same issue is reproducing in

the table control, "names" and its background, the ratio is 3.658:1
ExpenseItIntro->Expense report for-Color contrast is failing for table and overall background.
ExpenseItIntro->Expense report for-Color contrast for letters in the table and the blue background is failing.
DataBindingDemo->List of products-color contrast for the given highlighted colors is not compliant to the existing ratio.
### Expected Result:
The color contrast ration between title "View expense report" and background should be greater than or equal to 4.5:1"

### User Impact:
Low Vision users may find difficulty in reading the text.

**Comment by merriemcgaw** [comment]
@Blessymanna please provide an image of this problem?

**Comment by ayushverma-ms** [comment]
This issue is fixed. PR - https://github.com/microsoft/WPF-Samples/pull/399
